src/modules/script.cpp

```diff
@@ -25,7 +26,12 @@ namespace modules {
     // Setup formatting
     m_formatter->add(DEFAULT_FORMAT, TAG_LABEL, {TAG_LABEL});
     if (m_formatter->has(TAG_LABEL)) {
-      m_label = load_optional_label(m_conf, name(), "label", "%output%");
+      m_label = load_optional_label(m_conf, name(), TAG_LABEL, "%output%");
+    }
+
+    m_formatter->add(FORMAT_FAIL, TAG_LABEL_FAIL, {TAG_LABEL_FAIL});
```

Comment:
There is an `add_optional` function in `m_formatter` that makes it easier to have optional formats. Have a look at how the CPU module does this:

https://github.com/polybar/polybar/blob/ab915fb724546e7abfcb35cbaf863ce2bca613ec/src/modules/cpu.cpp#L23-L24

https://github.com/polybar/polybar/blob/ab915fb724546e7abfcb35cbaf863ce2bca613ec/src/modules/cpu.cpp#L94-L100